### ISSUE TITLE
[QUESTION] Skipping examples with NaN loss

### ISSUE THREAD

**Issue by gahdritz** [microsoft/DeepSpeed#1496]
I'm trying to train a model on multiple GPUs with fp16 mixed precision and DeepSpeed + PyTorch Lightning. Certain pathological examples in the training set cause activations to overflow and ultimately the loss to become NaN. Is there any way to skip the backward pass for such examples? Alternatively, how is this problem usually addressed? My current hack is just to zero the loss in these cases, but that interferes with average loss computation and is generally inelegant.
